### PR TITLE
reworked Current Status -> SciPy Today

### PR DESCRIPTION
**Pull request by mdhaber**
The "Current Status" section didn't seem to be complete. I thought it made more sense to put a "SciPy Today" section after "SciPy Matures", and this is the information I thought it should include to follow all the history in the preceeding sections.

### CODE REVIEW

**Comment by tylerjereddy** [comment]
I'm -0.5 on using logos for the timeline. I think this new diagram with project names is helpful, but I'm not convinced the bulky logos are particularly helpful to conveying the point of the diagram.

**Comment by mdhaber** [comment]
Less bulky any better?
scipy_timeline
I see the point, though. I guess I'm not sure whether the figure is necessary in the first place. But I figured if it's going to take up space it should look interesting, so I tried to make it look interesting.

**Comment by tylerjereddy** [comment]
I still think we can do just fine with the names for the node points. It can be made pretty with i.e., tikz nodes to add color / transparency / categories / whatever.

File: scipy-1.0/paper.tex [hunk 1]
```diff
@@ -649,6 +649,25 @@ \subsection*{SciPy Matures}
 community driven educational programs, meetups, and conferences
 called PyData.
 
+\subsection*{SciPy Today}
+At the time of writing, the SciPy library consists of nearly 
+600,000 lines of code organized in 16 subpackages. 
+Over 85,000 GitHub repositories and almost 5000 packages depend
+on SciPy. SciPy's GitHub repository attracts approximately 20,000 unique 
+visitors every two weeks, and more than 100 people contribute to a
```
Comment (tylerjereddy) [inline scipy-1.0/paper.tex]: This information is already in the `Maintainers and contributors` subsection

File: scipy-1.0/paper.tex [hunk 2]
```diff
@@ -649,6 +649,25 @@ \subsection*{SciPy Matures}
 community driven educational programs, meetups, and conferences
 called PyData.
 
+\subsection*{SciPy Today}
+At the time of writing, the SciPy library consists of nearly 
+600,000 lines of code organized in 16 subpackages. 
+Over 85,000 GitHub repositories and almost 5000 packages depend
+on SciPy. SciPy's GitHub repository attracts approximately 20,000 unique
```
Comment (tylerjereddy) [inline scipy-1.0/paper.tex]: this information is already in the `Impact  now`subsection

**Comment by mdhaber** [comment]
I realized after writing that the information was already elsewhere. But:
a) with SciPy Begins and SciPy Matures sections, it would be awkward to end this section without a SciPy Today (or "Current Status") section, and
b) The original content:
```
At the time of writing, SciPy produces a new release
approximately every six months, and some of the major
feature highlights from the three years preceding
SciPy 1.0 are discussed in a technical improvements
section below. Some longer-term milestone events are
highlighted in Figure~\ref{fig:timeline}.
```
has very little information about the current status of the project.

**Comment by tylerjereddy** [comment]
The paper is about SciPy 1.0 so I'd be fine with "Current Status" being fairly loose / mostly about 1.0 at the introduction level. Maybe a renaming of some sort is in order, but I don't think we want to set the stage for a paper with substantive content about SciPy > 1.0.

Maybe just mention those few bonus sections later on that sneak in small bits of material about things after 1.0, and emphasize that the manuscript is about the 1.0 release point.

But yeah, let others chime in & take it from there.

**Comment by rgommers** [comment]
I'm fine with the way it looks now - perhaps just merge and move on? We need to get this over the line at some point .....

**Comment by ilayn** [comment]
I agree with @rgommers if you want something really quick as a placeholder I typed the following 

```latex
\begin{tikzpicture}[ultra thick,line cap=round,scale=1.2,]
\draw (1,0) -- (17,0);
\foreach\x[count=\xi] in {1,5,7,8,10,11,12,13,15,17}{\draw (\x,0) -- +(0,\ifodd\xi-\fi1) coordinate (n-\x) {};}
\node[anchor=north, align=center] at (n-1) {Scipy 0.1 \\ released};
\node[anchor=south, align=center] at (n-5) {Transition\\ to NumPy};
\node[anchor=north, align=center] at (n-7) {Creation of \\ SciKits};
\node[anchor=south, align=center] at (n-8) {First\\ Cython code\\ \texttt{scipy.spatial}};
\node[anchor=north, align=center] at (n-10) {6-month\\ release\\ cycle}; 
\node[anchor=south, align=center] at (n-11) {Development\\ moves\\to GitHub};
\node[anchor=north, align=center,font=\ttfamily] at (n-12) {sparse.csgraph\\optimize.minimize};
\node[anchor=south, align=center] at (n-13) {Continuous\\Integration\\with TravisCI};
\node[anchor=north, align=center] at (n-15) {Cython\\Interface for\\BLAS/LAPACK};
\node[anchor=south, align=center] at (n-17) {Continuous\\Integration\\with TravisCI};
\end{tikzpicture}
```

image


So we can keep track of it. But we might want to get to the first draft before momentum is drained. So please go ahead and merge it and we can polish the appearance later in great detail.

**Comment by mdhaber** [comment]
I normally wouldn't self-merge but I take the past two comments as instruction to do so.

@ilayn thanks for the figure code but there is already a placeholder, so I'm going to leave as-is for now. Also, bulkiness was the stated concern for the figure, but the aspect ratio of these two timelines are nearly identical. Since you are planning to polish figures later, I'll leave it to then.

**Pull request merged**